app/controllers/photos_controller.rb

```diff
@@ -1,6 +1,9 @@
 class PhotosController < ApplicationController
   before_action :set_photo, only: %i[ show edit update destroy ]
   before_action :ensure_current_user_is_owner, only: [:destroy, :update, :edit]
```

Comment:
We can now delete this, we are replacing these logics with Pundit. It's a way to organize authorization.

Reply:
That is good to know that because of pundit we no longer need those logics because I somehow thought it was still needed in some way and Pundit built on top of it which is not the case.